src/legacy/status_im/qr_scanner/core.cljs

```diff
@@ -78,8 +78,8 @@
 (rf/defn handle-eip681
   [cofx data]
   (rf/merge cofx
-            {:dispatch [:wallet-legacy/parse-eip681-uri-and-resolve-ens data]}
-            (navigation/change-tab :wallet-stack)
+            {:dispatch   [:wallet-legacy/parse-eip681-uri-and-resolve-ens data]
+             :dispatch-n [[:shell/change-tab :wallet-stack]]}
```

Comment:
Could we use just one `dispatch-n` with `[:wallet-legacy/parse-eip681-uri-and-resolve-ens data]` preceding `[:shell/change-tab :wallet-stack]` ?